### PR TITLE
7524 logger for swifmailer extension

### PR DESCRIPTION
**Pull request by sdlins**
I think it is a simple way to solve the swiftmailer logger problem.
#7524

### CODE REVIEW

File: extensions/swiftmailer/Mailer.php [hunk 1]
```diff
@@ -171,6 +176,10 @@ protected function createTransport(array $config)
                 if (is_array($plugin) && isset($plugin['class'])) {
                     $plugin = $this->createSwiftObject($plugin);
                 }
+                $interfaces = class_implements(get_class($plugin));
```
Comment (samdark) [inline extensions/swiftmailer/Mailer.php]: Can we use `$plugin instanceof \Swift_Plugins_Logger` instead?

**Comment by sidtj** [comment]
> Can we use $plugin instanceof \Swift_Plugins_Logger instead?

Thanks guys! Much SIMPLER!
I have already done... will PR.

**Comment by sidtj** [comment]
I also fixed the scope of $_logger to protected.

**Comment by sidtj** [comment]
It is done. swiftmailerTest passing localy.

**Comment by klimov-paul** [comment]
Closed in favor of https://github.com/yiisoft/yii2-swiftmailer/pull/5

**Pull request closed without merging**